components/tools/OmeroWeb/test/integration/test_api_containers.py

```diff
@@ -156,6 +171,22 @@ def screen_plates(self, user1):
 
         screen = get_update_service(user1).saveAndReturnObject(screen)
         plate = get_update_service(user1).saveAndReturnObject(plate)
+
+        # Add well to first plate
```

Comment:
I will do in both cases to be on the safe side
i.e. with plate acquisitions and without